packages/ckeditor5-ui/src/index.ts

```diff
@@ -75,6 +75,8 @@ export { default as BlockToolbar } from './toolbar/block/blocktoolbar';
 export { default as View, type UIViewRenderEvent } from './view';
 export { default as ViewCollection } from './viewcollection';
 
-export { default as ColorPaletteIcon } from '../theme/icons/color-palette.svg';
+import { default as colorPalette } from '../theme/icons/color-palette.svg';
+
+export const ColorPaletteIcon = colorPalette;
```

Comment:
Is it consistent with other icons that this one is UpperCased?

Reply:
Actually, I can't see any other icon exported this way:

![](https://github.com/ckeditor/ckeditor5/assets/156149/3b02940a-1902-4978-bdc1-072d77567f3f)

Reply:
Other icons (coming from our premium features) are exported lowercased so that's what we do here as well.